src/justintime/plots/content/01_home_plot.py

```diff
@@ -34,15 +34,16 @@ def init_callbacks(dash_app, storage, plot_id,theme):
     @dash_app.callback(
         Output(plot_id, "children"),
         Input("90_plot_button_ctrl", "n_clicks"),
+        Input('01_home_plot', 'style'),
```

Comment:
Could you explain why the new `Inputs`: `home_plot` and `trigger_record_select_ctrl`.
There was a reason not to trigger the plotting upon the selection of the trigger record, which is that the user should choose the setting first and explicitly plus on purpose, since plotting can be a heavy operation.